src/components/BodySection.js

```diff
@@ -87,9 +87,18 @@ export default function BodySection(props) {
           <path d={frontMuscles.body} />
           <path d={frontMuscles.body} transform={mirror} />
         </g>
+        <g
+          className="frontFace"
+          id="FrontFace"
+          stroke="black"
+          fill="white"

```

Comment:
Added a strokeWidth of 2 which I think helps the body pop a bit. Think we should change it.

<img width="604" alt="image" src="https://user-images.githubusercontent.com/66337956/224589317-1087b490-ea2b-4e70-b210-437343b4661e.png">

Reply:
Can push the changes if you like it. Otherwise PR looks good.

Reply:
Yes please!